Sources/Accord.Math/Optimization/QuadraticObjectiveFunction.cs

```diff
@@ -127,20 +145,23 @@ namespace Accord.Math.Optimization
     ///   double value = solver.Value;
     ///   double[] solutions = solver.Solution;
     /// </code>
+    /// 
+    /// <para>
+    ///   Sometimes it easiest to compose quadratic functions as linear

```

Comment:
Is there a typo here? Sorry I am not a native English speaker, so I thought it was better to ask :-)

Reply:
There was indeed. Thanks for spotting. I have fixed the typo and pushed. BTW - if you want a native speaker to review your arXiv article I am more than happy to. However, to be completely clear, your English seems absolutely faultless - and I make typos :) - so I'm not sure I see the need!